### ISSUE TITLE
Help needed with getting MLPlan running

### ISSUE THREAD

**Issue by Perevalov** [starlibs/AILibs#273]
I tried to create a simple Java Maven project in IntelliJ Idea and added the following dependency to the `pom.xml`,

```xml
<dependency>
  <groupId>ai.libs</groupId>
  <artifactId>mlplan-full</artifactId>
  <version>0.2.5</version>
</dependency>
```

however, when trying to run the project, the following error arises:
![image](https://user-images.githubusercontent.com/16652575/232476261-e3209f8b-f027-410d-af64-9c1622bfc7eb.png)

Could you give any guidance on how to proceed further?

I'm using Java 17.0.6 and Maven 3.8.8

**Comment by mwever** [comment]
Hi @Perevalov ,
it looks like the jitpack repository is not available for your build?

In gradle you need to declare the repositories artifacts can be pulled from like this:
```
	repositories {
		mavenCentral()
		mavenLocal()
		maven { url "https://jitpack.io" }
		maven { url "https://clojars.org/repo/" }
		maven { url "https://plugins.gradle.org/m2/" }
	}
```

or in maven like this:
```
	<repositories>
		<repository>
		    <id>jitpack.io</id>
		    <url>https://jitpack.io</url>
		</repository>
	</repositories>
```

Can you please make sure that all necessary repositories are availble to your build?

**Comment by Perevalov** [comment]
Hi @mwever, thanks, it builds now. I will proceed with exploring the framework. Do you have any projects/code examples apart from the https://starlibs.github.io/AILibs/projects/mlplan/?

**Comment by mwever** [comment]
Yes, actually almost every sub-project has a source folder for examples. In particular the sub-projects for ML-Plan variants (sklear, WEKA, MEKA, etc.) do come with different examples. Structurally they are almost identical as only a builder is instantiated and then ML-Plan is configured via the builder, and eventually ML-Plan is called to return a pipeline. You will also find some examples providing a graphical interface to oberserve the progress of the search.
